### DISCUSSION TITLE
Object detector with MLKit

### DISCUSSION THREAD

**Discussion by ghost** [sceneview/sceneform-android#254] [Q&A]
Hello everyone, i am trying find a way to make android app with feature:
- first, detect mlkit object
- then draw 3D model animation on ml object detected
So, does this library has support my requirement?

**Comment by ThomasGorisse** [comment]
You will have to do the ML part on your own and then add your 3D to Sceneform.

Have a look in here: https://github.com/googlesamples/arcore-ml-sample
It's a great ARCore project around ML made by @devbridie

**Reply by ghost** [reply]
@ThomasGorisse hello bro, can i ask you a few questions?
- In augmented image demo project: What diffirent between Video.glb and Rabbit.glb? How do i know whether to use file as Video.glb or Rabbit.glb for my project?
- Can i use other formats besides *.glb format? If yes, how to use it?
